kubeflow/instance/kustomize/kubeflow-apps/kustomization.yaml

```diff
@@ -5,5 +5,5 @@ patchesStrategicMerge:
 - default-install-config.yaml
 - pipeline-minio-install-config.yaml
 resources:
-- ../../../upstream/manifests/stacks/gcp # {"type":"string","x-kustomize":{"setBy":"kpt","partialSetters":[{"name":"kustomize_manifests_path","value":"../../../upstream/manifests"}]}}
```

Comment:
not very related to this PR, but shall we add gcp stacks back to manifests, I'm making some gcp specific components and I think a gcp stacks is needed, we cannot compose all of them in instance/ folder